qdrant_client/connection.py

```
@@ -236,19 +236,34 @@ async def intercept_call(
+        "certificate_chain": None,
+    }
+
     if options is None:
-        return default_options
+        return default_options, ssl_creds_options
+
+    _options: list[tuple[str, Any]] = []
+    for option_name, option_value in options.items():
+        if option_name.removeprefix("grpc.") in ssl_creds_options and option_value:
+            if not isinstance(option_value, bytes):
+                raise TypeError(f"{option_name} must be bytes, got {type(option_value).__name__}")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)